modules/backing-up-red-hat-quay.adoc

```diff
@@ -0,0 +1,251 @@
+-n <quay-namespace>[[backing-up-red-hat-quay]]
```

Comment:
Does line 1 is the typo?  From review page, the format is weird.
If it is desired format, please ignore this comment.